dev/CommonStyles/CornerRadius_themeresources.xaml

```diff
@@ -5,16 +5,16 @@
     xmlns:primitives="using:Microsoft.UI.Xaml.Controls.Primitives">
     <ResourceDictionary.ThemeDictionaries>
         <ResourceDictionary x:Key="Default">
-            <CornerRadius x:Key="ControlCornerRadius">2,2,2,2</CornerRadius>
-            <CornerRadius x:Key="OverlayCornerRadius">4,4,4,4</CornerRadius>
+            <CornerRadius x:Key="ControlCornerRadius">4,4,4,4</CornerRadius>
```

Comment:
Out of interest, what has changed for MS Design since the discussion in https://github.com/microsoft/microsoft-ui-xaml/issues/524 leading to this doubling of the corner radii?

Reply:
Please don't

Reply:
@Felix-Dev , I am not sure what part of the discussion you are pointing out of the #524, but nothing has fundamentally changed in terms of how we work. We continue to collaborate with design teams across the company.

Reply:
@chigy I was only interested in *why* the (overlay) control corner radii are being doubled as part of this PR (so the motivation behind this change) when #524 settled on 2px for control corner radii and 4px for overlay control corner radii (like flyouts). My question isn't about how the design teams at MS work or collaborate internally.

Reply:
@Felix-Dev , we had overlay corner doubling of the control corner in our previous design. We are merely increasing the control corner to 4px which makes overlay to become 8px. Nothing changed other than the value increased globally?

Reply:
I was more looking for an explanation like "customers felt 2px rounded corners were too subtle" or "MS Design felt 2px corner radii didn't fit too nicely with the rest of the corner radii seen in the web/mobile space". Anyway, I'll leave it at that.

Reply:
I agree with @Felix-Dev Here. He is simply asking why the pixel values changed. What is the reasoning behind it? As app developers we need to know so we can evaluate our own updates to match changes like this.

Reply:
To copy material is the right answer. 2px is the best choice since I other platforms rounding too much and causing a distraction.